packages/babel-core/README.md

```diff
@@ -123,6 +123,78 @@ const parsedAst = babylon.parse(sourceCode, { allowReturnOutsideFunction: true }
 const { code, map, ast } = babel.transformFromAstSync(parsedAst, sourceCode, options);
 ```
 
+## babel.parse(code: string, [options?](#options): Object)
```

Comment:
Does this return the AST or something else? Maybe we can add return types so I don't have to look up the code to see what this does. :)

Reply:
Sure, I'll add a note. Just realized we didn't document it when we actually landed that PR.